test/e2e/drc_crud/drc/framework.go

```diff
@@ -99,20 +99,6 @@ func (f *Framework) CreateRedisCluster(instance *redisv1alpha1.DistributedRedisC
 	return err
 }
 
-// CreateRedisClusterBackup creates a RedisClusterBackup in test namespace
-func (f *Framework) CreateRedisClusterBackup(instance *redisv1alpha1.RedisClusterBackup) error {
```

Comment:
I'd comment this out and add a comment above it if it's not workting. We still bave RedisClusterBackup in our code base.